spec/spec_helper.rb

```diff
@@ -151,24 +120,19 @@ def main_app
     c.syntax = :expect
   end
 
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = File.expand_path("../fixtures", __FILE__)
-
   config.use_transactional_fixtures = false
 
   config.before :suite do
+    Hyrax::RedisEventStore.instance.redis.flushdb
```

Comment:
do both supported set up methods (docker and ~boring~ classic) make it clear that redis should be separate between test and dev?

Reply:
i'm not sure. either way this rewrite makes us flush the redis db less frequently. the old setup ran it every test!

Reply:
sounds like no harm then =-)